tiingo/api.py

```diff
@@ -108,17 +108,20 @@ def list_tickers(self, assetType):
         raw_csv = get_buffer_from_zipfile(zipdata, 'supported_tickers.csv')
         reader = csv.DictReader(raw_csv)
 
+        if not len(assetTypes):
+            return [row for row in reader]
+
         return [row for row in reader
-                if row.get('assetType') == assetType]
+                if row.get('assetType') in assetTypes]
```

Comment:
I think we internally may want to convert `assetTypes` into a python Set, otherwise we will repeat an `O(n)` operation for every row, whereas if we make this into a Set, then set membership testing will happen in `O(1)` for each row. Let me know if you have further questions about what I mean.

Reply:
:+1: